### PR TITLE
MULE-17921: Support Platform Managed OAuth tokens

### PR DESCRIPTION
**Pull request by marianogonzalez**

### CODE REVIEW

File: services-all/pom.xml [hunk 1]
```diff
@@ -27,7 +27,19 @@
                 <groupId>org.mule.services</groupId>
                 <artifactId>mule-service-http</artifactId>  
                 <version>${muleHttpServiceVersion}</version>
-            </dependency>  	
+            </dependency>
+            <dependency>
+                <groupId>org.mule.services</groupId>
+                <artifactId>mule-service-oauth</artifactId>
+                <version>${muleOAuthServiceVersion}</version>
```
Comment (elrodro83) [inline services-all/pom.xml]: remove the versions from the dependencies in the `dependencies` section (http too now that you're at ti)

File: services-all/pom.xml [hunk 2]
```diff
@@ -27,7 +27,19 @@
...
+            </dependency>
+            <dependency>
+                <groupId>org.mule.services</groupId>
+                <artifactId>mule-service-oauth</artifactId>
+                <version>${muleOAuthServiceVersion}</version>
+                <type>test-jar</type>
```
Comment (elrodro83) [inline services-all/pom.xml]: why is this needed here?

Comment (marianogonzalez) [reply services-all/pom.xml]: for the EE version
